File: j2k/tests/testData/fileOrElement/tryWithResource/WithReturnAtEnd.java
//file
import java.io.*;

public class C {
    int foo() {
        try(InputStream stream = new FileInputStream("foo")) {
            // reading something
            int c = stream.read();
            return c;
        }
        catch (IOException e) {
            System.out.println(e);
            return -1;
        }
    }
}

File: j2k/tests/testData/fileOrElement/tryWithResource/WithReturnAtEnd.kt
// ERROR: 'return' is not allowed here
// ERROR: A 'return' expression required in a function with a block body ('{...}')
import java.io.*

public class C {
    fun foo(): Int {
        try {
            ByteArrayInputStream(ByteArray(10)).use { stream ->
                // reading something
                val c = stream.read()
                return c
            }
        } catch (e: IOException) {
            System.out.println(e)
            return -1
        }

    }
}